docs/core/porting/third-party-deps.md

```diff
@@ -104,6 +104,12 @@ To suppress the warning by editing the project file, find the `PackageReference`
 
 For more information on how to suppress compiler warnings in Visual Studio, see [Suppressing warnings for NuGet packages](/visualstudio/ide/how-to-suppress-compiler-warnings#suppressing-warnings-for-nuget-packages).
 
+### Port your packages to `PackageReference`
+
+.NET Core uses `PackageReference` to specify package dependencies. If you are using `packages.config` to specify your packages, you will need to convert over to `PackageReference`.
+
+You can learn more at [Migrate from packages.config to PackageReference](../../../nuget/reference/migrate-packages-config-to-package-reference).
```

Comment:
`For more information, see [Migrate from packages.config to PackageReference](/nuget/reference/migrate-packages-config-to-package-reference).`